python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -84,9 +204,7 @@ def _on_before_paint(self, widget, model_index, style_options):
         # v004 (2014-02-21 12:34)
 
         header_str = ""
-
-        if sg_item.get("version_number"):
-            header_str += "<b style='color:#2C93E2'>Version %03d</b>" % sg_item.get("version_number")
+        header_str += "<b style='color:#2C93E2'>Version %03d</b>" % (sg_item.get("version_number") or 0)
```

Comment:
Yes, but what I getting at is that I would have thought like Alan that sg_data.get("version_number", 0) would have worked. It's confusing to have None stored in the dictionary for that entry. That's my take on it anyway.